### ISSUE TITLE
How do I Add comments to my app?

### ISSUE THREAD

**Issue by dangdv** [Wizcorp/phonegap-facebook-plugin#828]
<div id="fb-root"></div>

<script>(function(d, s, id) {
  var js, fjs = d.getElementsByTagName(s)[0];
  if (d.getElementById(id)) return;
  js = d.createElement(s); js.id = id;
  js.src = "//connect.facebook.net/vi_VN/sdk.js#xfbml=1&appId='MY ID'&version=v2.0";
  fjs.parentNode.insertBefore(js, fjs);
}(document, 'script', 'facebook-jssdk'));
</script>

<div class="fb-comments" data-href="http://developers.facebook.com/docs/plugins/comments/" data-numposts="5" data-colorscheme="light"></div>

https://developers.facebook.com/docs/plugins/comments?locale=vi_VN
IT IS WORKING ON THE WEB , BUT I DONT KNOW HOW TO DO IT ON MY CORDOVA APP

**Comment by felquis** [comment]
I don't think it's possible inside a hybrid app, but maybe it'll work in a responsive web site.

**Comment by yurik94** [comment]
I really need it, is this possible now??

**Comment by ervaibhavsinghal** [comment]
@dangdv @yurik94 hey guys... do you found something else to do this task??

**Comment by yurik94** [comment]
@ervaibhavsinghal Nope... I had to remove the functionality of fb comments on my app until I find a solution 👎
